web/client/utils/__tests__/GeostoreUtils-test.js

```diff
@@ -99,92 +104,42 @@ describe('GeostoreUtils', () => {
             name: 'Context'
         })).toEqual({ items: [{ type: 'icon', glyph: 'context', tooltipId: 'resourcesCatalog.mapUsesContext', tooltipParams: { contextName: 'Context' } }] });
     });
-    it('computePendingChanges', () => {
```

Comment:
all these removed cases must be rewritten using computeSaveResource and computePendingChanges